inspirehep/modules/hal/config.py

```diff
@@ -20,10 +20,46 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-"""HAL module configuration."""
+"""HAL Configuration."""
 
 from __future__ import absolute_import, division, print_function
 
 
-HAL_EXPORT_TEMPLATE = 'hal_tei.xml'
-"""Filename of the base TEI template in the template environment."""
+#
+# Configuration used when converting to TEI.
+#
+
+HAL_DOMAIN_MAPPING = {
+    'Accelerators': 'phys.phys.phys-acc-ph',
+    'Astrophysics': 'phys.astr',
+    'Computing': 'info',
+    'Data Analysis and Statistics': 'phys.phys.phys-data-an',
+    'Experiment-HEP': 'phys.hexp',
+    'Experiment-Nucl': 'phys.nexp',
+    'General Physics': 'phys.phys.phys-gen-ph',
+    'Gravitation and Cosmology': 'phys.grqc',
+    'Instrumentation': 'phys.phys.phys-ins-det',
+    'Lattice': 'phys.hlat',
+    'Math and Math Physics': 'math',
+    'Other': 'phys',
+    'Phenomenology-HEP': 'phys.hphe',
+    'Theory-HEP': 'phys.hthe',
+    'Theory-Nucl': 'phys.nucl',
+}
+"""Mapping used when converting from INSPIRE categories to HAL domains."""
+
+
+#
+# Configuration used when connecting to HAL.
+#
+
+HAL_COL_IRI = 'https://api-preprod.archives-ouvertes.fr/sword/hal'
+HAL_EDIT_IRI = 'https://api-preprod.archives-ouvertes.fr/sword/'
+"""IRIs used by the SWORD protocol when connecting to HAL."""
+
+HAL_USER_NAME = 'hal_user_name'
+HAL_USER_PASS = 'hal_user_pass'
+"""INSPIRE credentials on HAL."""
+
+HAL_IGNORE_CERTIFICATES = False
+"""Whether to check certificates when connecting to HAL."""
```

Comment:
If you remember, this was needed when one of the servers in their staging environment was misconfigured, but they since fixed the bug. I chose to preserve this extra knob just to be a little more configurable, but they can be pulled in the previous `APP` variable.